Dockerfile

```diff
@@ -19,6 +19,9 @@ RUN npm install -g yarn
 # download the application dependencies
 RUN yarn install
 
+# little fix
+RUN npx browserslist --update-db
+
```

Comment:
Remove this since this issue is fixed now

Reply:
Fixed